combat-trainer.lic

```diff
@@ -255,10 +260,10 @@ class LootProcess
       check_skinning(DRRoom.dead_npcs.first, game_state)
     end
     while 'and get ready to search it' == bput('loot', 'You search', 'I could not find what you were referring to', 'and get ready to search it')
+      @loot_timer = Time.now
```

Comment:
The idea was to just update the timer if a looting is actually performed to act as a stopgap between the first check in `dispose_body` being `return if DRRoom.dead_npcs.empty?`